### ISSUE TITLE
make csx tools load multiple scan_ids

### ISSUE THREAD

**Issue by mpmdean** [NSLS-II-CSX/csxtools#64]
Should csxtools be able to handle passing more than I scanid to db as below? 
```
from databroker import DataBroker as db
from csxtools.utils import get_fastccd_images

slicerator = get_fastccd_images(db[108225, 108226])

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-5adf5db257b5> in <module>()
      2 from csxtools.utils import get_fastccd_images
      3 
----> 4 slicerator = get_fastccd_images(db[108225, 108226])

/opt/conda_envs/analysis-2018-2.1/lib/python3.6/site-packages/csxtools/utils.py in get_fastccd_images(light_header, dark_headers, flat, gain, tag, roi)
    119         logger.info("Computed dark images in %.3f seconds", ttime.time() - t)
    120 
--> 121     events = _get_images(light_header, tag, roi)
    122 
    123     # Ok, so lets return a pims pipeline which does the image conversion

/opt/conda_envs/analysis-2018-2.1/lib/python3.6/site-packages/csxtools/utils.py in _get_images(header, tag, roi)
    179 def _get_images(header, tag, roi=None):
    180     t = ttime.time()
--> 181     images = header.db.get_images(header, tag)
    182     t = ttime.time() - t
    183     logger.info("Took %.3f seconds to read data using get_images", t)

AttributeError: 'list' object has no attribute 'db'
```

**Comment by licode** [comment]
I think this is implemented (https://github.com/NSLS-II-CSX/csxtools/commit/8a4e8ca17369e64136e6fdad9f57cb6ba97aee5f), but we didn't release the new version yet. I am going to add a new version number, and try to update csxtools in the current conda env.

**Comment by danielballan** [comment]
This fix was released in 0.1.14 and deployed.

**Comment by licode** [comment]
We didn't deploy it yet. I will test more before deploying it. It looks like we don't have enough test in this repo.

**Comment by mpmdean** [comment]
Thanks.

**Comment by mpmdean** [comment]
I just checked this on CSX srv1 (current) and CSX srv2 (current) kernels and it returns a similar error. Maybe we can re-open this until it's deployed?

**Comment by mpmdean** [comment]
Hi @danielballan 
This is fixed in v0.1.15, which one can install into a custom environment, but the current kernels at CSX feature v0.1.13. 

 
![Screen Shot 2020-04-29 at 11 16 17 AM](https://user-images.githubusercontent.com/699991/80613703-9eb29b00-8a0b-11ea-817d-e0749727f748.png)
![Screen Shot 2020-04-29 at 11 15 36 AM](https://user-images.githubusercontent.com/699991/80613705-9f4b3180-8a0b-11ea-8a4f-344d97ea8ae1.png)
